dom.bs

```diff
@@ -727,16 +729,15 @@ inherits from the {{Event}} interface.
 
  <dt><code><var>event</var> . <a method for=Event lt="preventDefault()">preventDefault</a>()</code>
  <dd>If invoked when the
- {{Event/cancelable}} attribute value is true,
+ {{Event/cancelable}} attribute value is true, and while executing a listener
+ for the <var>event</var> with {{EventListenerOptions/passive}} set to false,
  signals to the operation that caused <var>event</var> to be
  <a>dispatched</a> that it needs to be
  canceled.
 
  <dt><code><var>event</var> . {{Event/defaultPrevented}}</code>
  <dd>Returns true if
- {{Event/preventDefault()}} was invoked
- while the {{Event/cancelable}} attribute
- value is true, and false otherwise.
+ {{Event/preventDefault()}} was used successfully to indicate cancellation.
```

Comment:
(Adding more nitpicky comments. Feel free to ignore for now.) s/used/invoked/ We don't want to omit mentioning when it returns false. The original text did that correctly.